src/requests/models.py

```diff
@@ -663,10 +665,10 @@ def __init__(self):
         #: Integer Code of responded HTTP Status, e.g. 404 or 200.
         self.status_code = None
 
-        #: Case-insensitive Dictionary of Response Headers.
+        #: Case-insensitive Dictionary (email.EmailMessage) of Response Headers.
```

Comment:
This is incorrect for documentation linking (if that's what you're going for)

Reply:
I am not particularly familiar with the documentation formatting. It does provide the class name that can be looked up but does not do much else if there are additional things that could be done.